resotocore/resotocore/config/config_handler_service.py

```diff
@@ -125,11 +145,46 @@ async def update_configs_model(self, kinds: List[Kind]) -> Model:
         return updated
 
     async def config_yaml(self, cfg_id: ConfigId, revision: bool = False) -> Optional[str]:
-        config = await self.get_config(cfg_id)
+        config = await self.get_config(cfg_id, apply_overrides=False)
         if config:
             model = await self.get_configs_model()
 
+            # returns the overridden config with comments about the changes
+            def overridden_parts(existing: JsonElement, update: JsonElement) -> JsonElement:
+                if isinstance(update, dict):
+                    return {
+                        key: overridden_parts(
+                            existing.get(key) if isinstance(existing, dict) else existing, update[key]
+                        )
+                        for key in update.keys()
+                    }
+                else:
+
+                    def mkstr(val: Any) -> str:
+                        return f'[{", ".join(val)}]' if isinstance(val, list) else str(val)
+
+                    if existing is None:
+                        return f"{mkstr(update)} - new value"
+
+                    return f"{mkstr(update)} - previously was {mkstr(existing)}"
+
             yaml_str = ""
+
+            if self.core_config.overrides:
+                yaml_str += (
```

Comment:
That puts the overriding statement on top. Didn't we want to have it on the property level?

Reply:
That's a bit more complicated and currently being worked on by me.